lib/features/family/features/reflect/presentation/pages/heart_success_screen.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app/core/enums/amplitude_events.dart';
+import 'package:givt_app/features/family/features/giving_flow/collectgroup_details/cubit/collectgroup_details_cubit.dart';
+import 'package:givt_app/features/family/features/profiles/cubit/profiles_cubit.dart';
+import 'package:givt_app/features/family/helpers/vibrator.dart';
+import 'package:givt_app/features/family/shared/design/components/actions/fun_button.dart';
+import 'package:givt_app/features/family/shared/widgets/buttons/back_home_button.dart';
+import 'package:givt_app/features/family/shared/widgets/buttons/switch_profile_success_button.dart';
+import 'package:givt_app/features/family/shared/widgets/texts/shared_texts.dart';
+import 'package:givt_app/features/family/utils/family_app_theme.dart';
+import 'package:givt_app/shared/models/analytics_event.dart';
+
+import 'package:lottie/lottie.dart';
+
+class HeartSuccessScreen extends StatefulWidget {
+  const HeartSuccessScreen({
+    super.key,
+    this.orgName,
+    this.onCustomSuccess,
+  });
+  final void Function()? onCustomSuccess;
+  final String? orgName;
+  @override
+  State<HeartSuccessScreen> createState() => _SuccessScreenState();
+}
+
+class _SuccessScreenState extends State<HeartSuccessScreen> {
+  @override
+  void initState() {
+    super.initState();
+
+    Vibrator.tryVibratePattern();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final organisation =
+        context.read<CollectGroupDetailsCubit>().state.collectgroup;
+    final profilesState = context.read<ProfilesCubit>().state;
+
+    return Scaffold(
+      appBar: AppBar(
+        toolbarHeight: 0,
+        backgroundColor: FamilyAppTheme.secondary98,
+        systemOverlayStyle: const SystemUiOverlayStyle(
+          statusBarColor: FamilyAppTheme.secondary98,
+        ),
+      ),
+      backgroundColor: FamilyAppTheme.secondary98,
+      body: SafeArea(
+        bottom: false,
+        child: Stack(
+          children: [
+            Positioned.fill(
+              child: Lottie.asset(
+                'assets/family/lotties/success_heart.json',
+                fit: BoxFit.fill,
+                alignment: Alignment.center,
+                width: double.infinity,
+              ),
+            ),
+            Container(
+              padding: const EdgeInsets.all(40),
+              width: double.infinity,
+              child: Column(
+                children: [
+                  TitleLargeText(
+                    organisation.thankYou ?? 'Awesome!',
+                    textAlign: TextAlign.center,
+                  ),
+                  Text(
```

Comment:
Just to be sure, did the design change? I would not expect the word donation here? In the "old" figma i see "Good luck in completing your act of service".

Reply:
right, i was looking at the lottie flow it doesnt have words, and thought the words were the same